rackcity/api/serializers/asset_serializers.py

```diff
@@ -392,4 +430,12 @@ def get_neighbor_assets(hostname, id, nodes, edges, change_plan=None):
                 )
         return nodes, edges
     except ObjectDoesNotExist:
-        return
\ No newline at end of file
+        return
+
+
+def get_datacenter_of_asset(asset):
+    if asset.rack:
+        datacenter = asset.rack.datacenter
+    if asset.chassis and asset.chassis.rack and asset.chassis.rack.datacenter:
```

Comment:
I think `and asset.chassis.rack.datacenter` is unnecessary but it's also correct so I don't mind it